shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -66,7 +67,10 @@
  * {@code Fragment}.
  */
 // TODO(mattcarroll): explain each call forwarded to Fragment (first requires resolution of PluginRegistry API).
-public class FlutterActivity extends FragmentActivity implements OnFirstFrameRenderedListener {
+public class FlutterActivity extends FragmentActivity
+    implements FlutterFragment.FlutterEngineProvider,
+    FlutterFragment.FlutterEngineConfigurator,
```

Comment:
I'm reading the PR sequentially so I'm making comments based on context from what's above the comment.

I'm getting the impression we were going to just push in and take away an activity from the platformviewcontroller. In that case, why is the activity involved? Couldn't the fragment just always get the activity on its own lifecycle events?

Reply:
I'm not sure what you're asking here. Can you rephrase?

Reply:
You were saying that this whole file's diff is unrelated to the platform view thing so disregard my comment.

Reply:
Talked offline, resolving old comments.